data-access/mandatees-decisions.ts

```diff
@@ -9,7 +9,10 @@ import {
 import { TERM_TYPE, sparqlEscapeTermValue } from '../util/sparql-escape';
 import { MANDATARIS_STATUS } from '../util/constants';
 
-const STAGING_GRAPH = 'http://mu.semte.ch/graphs/besluiten-consumed';
+export const TERM_STAGING_GRAPH = {
```

Comment:
I don't really get the use of this type in the TERM_STAGING_GRAPH

Reply:
Ah never mind, it is just so you can use the more general sparqlExcapeTermValue, right?